commit 987ab9a0c79fc3cc7aee901b4fb876ab1885434e (refs/changes/32/3432/3)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2018-02-28T14:52:33+00:00 (1 year, 7 months ago)
    
    OS-6654 allow brands to override CPU bursting setup
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/overlay/generic/usr/lib/brand/jcommon/statechange b/overlay/generic/usr/lib/brand/jcommon/statechange
index 5efd3e26..2678b0fa 100644
--- a/overlay/generic/usr/lib/brand/jcommon/statechange
+++ b/overlay/generic/usr/lib/brand/jcommon/statechange
@@ -632,6 +632,9 @@ setup_snapshots()
 #
 setup_cpu_baseline()
 {
+	# A brand can override the setup of bursting.
+	[ -n "$NO_BURSTING" ] && return
+
 	# If there is already a baseline, don't set one heuristically
 	curr_base=`prctl -P -n zone.cpu-baseline -i zone $ZONENAME | nawk '{
 		if ($2 == "privileged") print $3
